.github/workflows/versioning.yml

```diff
@@ -3,7 +3,7 @@ name: Version Bump
 on:
```

Comment:
Do we ever want to release from `master`? I don't believe it's something we currently want in our workflow? If not, we should probably add a check for that here. I have in the past accidentally released from `master` instead of `stable-vX`, because `master` is our default branch and I forgot to change the option in the release UI.

Reply:
No, I don't think we should ever release from `master`.

Let me find what's the best way to constraint this here.

Reply:
should be addressed by the new changes: learned much about GH workflows thanks to this one, especially to be able to bypass the default parallelism of actions and jobs

Reply:
Nice, we also use that to first test on baselines and only then frameworks too 👍